xscen/indicators.py

```diff
@@ -54,117 +60,7 @@ def load_xclim_module(filename, reload=False):
     return xc.build_indicator_module_from_yaml(filename)
 
 
-@parse_config
-def compute_indicators(
```

Comment:
Why move this away?

Reply:
Based on the list of top-level functions mentioned in #40, I figured it would make sense to place the most important things in their own file, so as not to have to exclude too much from `__all__`. Will revert.